packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -770,6 +773,10 @@ export default defineComponent({
 <style lang="scss">
 .resource-table {
   &-resource-wrapper {
+    &-limit-max-width {
+      max-width: 90%;
```

Comment:
does `calc(100% - var(--oc-space-medium))` work as well?

Reply:
Yes, that's actually better 👍  Thx!